backend/controllers/org.controller.ts

```diff
@@ -10,20 +41,47 @@ export const organizationTest = async (req: Request, res: Response) => {
 
 // Create a new Organization
 export const createNewOrganization = async (
-  req: Request,
+  // req: Request,
```

Comment:
Not sure why this is commented out, regardless please don't leave commented out code in the PR

Reply:
Forgot to remove that. Using the extended request instead